.eslintrc.js

```diff
@@ -144,6 +144,7 @@ module.exports = {
       files: ['*.tsx', '*.ts'],
       rules: {
         'react/require-default-props': 'off',
+        'react/destructuring-assignment': 'off',
```

Comment:
I found this doesn't seem to have any quality benefits, and I ended up needing to write less readable code. May as well just disable it.